docs/building.md

```diff
@@ -11,8 +11,9 @@ Connector App and examples of how programs can communicate with the app.
   Building under a different \*nix system, Mac OS or Windows should be possible
   too, though most probably will require more efforts.
 
-* The following tools should be present in the system: **bash**, **make**,
-  **curl**, **sed**, **mktemp**, **realpath**, **xxd**.
+* The following tools should be present in the system: **bash**, **make**
+  (version 4.3+ is recommended), **curl**, **sed**, **mktemp**, **realpath**,
```

Comment:
"recommended", or "required"?

Reply:
It's recommended, because the build will still work fine in a single-thread mode.

However, thinking more on this, perhaps we shouldn't depend on something that's so new that even the latest Ubuntu doesn't ship it by default... Also it might be even more problematic to set up for other developers (we know that some of them use Redhat).

So I'm thinking now if I should abandon this "compile stuff from sources" exercise and rather update our makefile to stop using this shiny new feature...